src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3997,6 +4121,130 @@ namespace ZoneTempPredictorCorrector {
 				SNLoad = 0.0;
 			}
 
+			//Hybrid modeling start: Added by Sang Hoon Lee May 2015
+			if (( HybridModelZone( ZoneNum ).InternalThermalMassCalc || HybridModelZone( ZoneNum ).InfiltrationCalc ) && !WarmupFlag && !DoingSizing ) {
+
+				Zone( ZoneNum ).ZoneMeasuredTemperature = GetCurrentScheduleValue( HybridModelZone( ZoneNum ).ZoneMeasuredTemperatureSchedulePtr );
+
+				// prepare start and end date for Hybrid Modeling
```

Comment:
1. Moved HybridStartDayOfYear and HybridEndDayOfYear computing to HybridModel module.

2. If hybrid modeling user's temperature data include a leap day, then the weather data should match match to this. The hybrid modeling calculation requires the actual weather data, the measured temperature data and weather data should match each year. A memo describing this added to HybridModel object in idd.